packages/mantine/src/components/modal-wizard/ModalWizard.tsx

```diff
@@ -72,25 +72,32 @@ interface ModalWizardProps {
      */
     modalProps?: any;
 
+    /**
+     * Classname for modal
+     */
+    modalClassName?: any;
+
     /**
      * Label for close button
      */
     closeButtonLabel?: string;
 
+    /**
+     * Props for progress bar
+     */
+    progressBarProps?: any;
```

Comment:
I'm sure there is a better type we can use here.

Reply:
Do we really want the progress bar to changes between each ModalWizards implementation? 🤔

Reply:
So currently there are already customizations possible for the progress bar, it is possible to hide progress bar for a ModalWizard implementation. And for our use case: we also want to have a different color (Teal) for the progress bar and a different size (lg)
<img width="699" alt="Screen Shot 2022-11-10 at 12 00 18 PM" src="https://user-images.githubusercontent.com/113382897/201159092-f5393a74-27bd-4680-ab39-d83293b65077.png">

Reply:
But should not ALL implementation of the progress bar in Modal wizard have the teal color? I just have to avoid having multiple colors used in different ModalWizard 😆 

If so, then lets "hardcode" it directly in the component

Reply:
I updated the PR to hardcode `teal` color into the progress bar, as well as the size :)